src/EnergyPlus/OutdoorAirUnit.cc

```diff
@@ -1211,372 +1215,51 @@ namespace OutdoorAirUnit {
 								ShowContinueError( "Verify that the value entered is intended and is consistent with other components." );
 							}
 						}
-
-						ReportSizingOutput( cFanTypes( OutAirUnit( OAUnitNum ).ExtFanType ), OutAirUnit( OAUnitNum ).ExtFanName, "Design Size Maximum Exhaust Air Flow Rate [m3/s]", ExtAirVolFlowDes, "User-Specified Maximum Exhaust Air Flow Rate [m3/s]", ExtAirVolFlowUser );
-						if ( DisplayExtraWarnings ) {
-							if ( ( std::abs( ExtAirVolFlowDes - ExtAirVolFlowUser ) / ExtAirVolFlowUser ) > AutoVsHardSizingThreshold ) {
-								ShowMessage( "SizeOutdoorAirUnit: Potential issue with equipment sizing for ZoneHVAC:OutdoorAirUnit " + cFanTypes( OutAirUnit( OAUnitNum ).SFanType ) + ' ' + OutAirUnit( OAUnitNum ).SFanName );
-								ShowContinueError( "User-Specified Maximum Exhaust Air Flow Rate of " + RoundSigDigits( ExtAirVolFlowUser, 5 ) + " [m3/s]" );
-								ShowContinueError( "differs from Design Size Maximum Exhaust Air Flow Rate of " + RoundSigDigits( ExtAirVolFlowDes, 5 ) + " [m3/s]" );
-								ShowContinueError( "This may, or may not, indicate mismatched component sizes." );
-								ShowContinueError( "Verify that the value entered is intended and is consistent with other components." );
-							}
-						}
 					}
 				}
 			}
 		}
 
```

Comment:
These next lines are just duplicate of what the coils will do. delete this stuff and instead mine the data from the coils.